microsoft/botframework-solutions
# [TypeScript] Add botskills to rush build flow

## Description
Add `botskills` project to rush build flow.
## Testing Steps
>NOTE: Run `npm install -g @microsoft/rush` to install the build tool globally.
1. Open a terminal in `./lib/typescript/`.
1. Run `npm run build` to install dependencies and build packages (This script runs `rush update && rush build` internally)
1. Run `npm run pack-local` to create the distributable packages for each project.
1. You should see the `.tgz` files in `./lib/typescript/outputpackages`.
